Gopkg.lock

```diff
@@ -2,76 +2,59 @@
 
 
 [[projects]]
-  digest = "1:0626d6801a7c1020d4586c9aa950638eab7b71fb7e4d8a18514f18b3dddf51be"
   name = "github.com/MysteriumNetwork/payments"
   packages = [
     "cli/helpers",
     "identity",
     "mysttoken/generated",
     "promises/generated",
     "registry",
-    "registry/generated",
+    "registry/generated"
   ]
-  pruneopts = "UT"
```

Comment:
@tadovas figured out that `dep` version 0.5.0 or higher required to avoid all these unnecessary changed. I'm on the way to add it to some description.